frontend/app/api.js

```diff
@@ -1,4 +1,5 @@
 const baseUrl = 'http://localhost:5040'
+const graphqlUrl = '/graphql'
```

Comment:
Out of scope for this PR, but I think we should use `SCREAMING_SNAKE_CASE` for constants. opinions? @jeppes @Tiseno

Reply:
hahah absolutely